src/Uno.Extensions.Templates/content/unoapp-extensions/azure/steps/desktop.yml

```diff
@@ -0,0 +1,81 @@
+parameters:
+- name: major
+  type: string
+  default: '0'  
+- name: minor
+  type: string
+  default: '0'
+- name: package_id
+  type: string
+  default: 'release_Desktop'
+- name: project_to_build
+  type: string
+  default: ''  
+- name: project_folder
+  type: string
+  default: ''    
+- name: project_platform
+  type: string
+  default: 'x86'  
```

Comment:
Should we default to x64 here? and/or generate both platforms?
cc @dansiegel @jeromelaban

Reply:
x64... does Apple actually make an x86 device anymore???